install_latest_blue_onyx.ps1

```diff
@@ -48,85 +48,103 @@ try {
     # --- 2. Download version.json into %TEMP% (always) ---
     $versionJsonFile = Join-Path $tempPath "version.json"
     $versionJsonUrl  = "https://github.com/xnorpx/blue-onyx/releases/latest/download/version.json"
-    Write-Host "Downloading version.json from $versionJsonUrl to $versionJsonFile..."
+    Write-Host "Downloading version.json from $versionJsonUrl to $versionJsonFile..." -ForegroundColor Green
```

Comment:
can you add:

if blue onyx exist in destination then:

.\blue_onyx.exe --version
blue-onyx 0.3.0

and if the version is the same as the latest then don't upgrade

Reply:
the script doesnt know what the destination path is yet (line 117) might have to shuffle code around to accomplish this.  let me try some things.

Reply:
added logic.